content/kubermatic/master/installation/add_seed_cluster/_index.en.md

```diff
@@ -0,0 +1,211 @@
+
+```yaml
+minio:
+  storeSize: '200Gi'
+  storageClass: hdd
+```
+
+It's also advisable to install the `s3-exporter` Helm chart, as it provides basic metrics about user cluster backups.
+
+#### Install Charts
+
+With this you can install the chart:
+
+```bash
+cd kubermatic-installer
+helm --tiller-namespace kubermatic upgrade --install --values /path/to/your/helm-values.yaml --namespace nodeport-proxy nodeport-proxy charts/nodeport-proxy/
+helm --tiller-namespace kubermatic upgrade --install --values /path/to/your/helm-values.yaml --namespace minio minio charts/minio/
+helm --tiller-namespace kubermatic upgrade --install --values /path/to/your/helm-values.yaml --namespace s3-exporter s3-exporter charts/s3-exporter/
+```
+
+### 3. Create Seed Resource
+
+To connect the new seed cluster with the master, you need to create a kubeconfig Secret and a Seed resource
+**in the master cluster**.
+
+{{% notice warning %}}
+Do not install the `kubermatic-operator` chart into seed clusters.
+{{% /notice %}}
+
+Make sure the kubeconfig contains static, long-lived credentials. Some cloud providers use custom authentication providers
+(like GKE using `gcloud` and EKS using `aws-iam-authenticator`). Those will not work in Kubermatic’s usecase because the
+required tools are not installed inside the cluster environment. You can use the `kubeconfig-serviceaccounts.sh` script from
+the kubermatic-installer repository to automatically create proper service accounts inside the seed cluster with static
+credentials:
+
+```bash
+cd kubermatic-installer
+./kubeconfig-service-accounts.sh mykubeconfig.yaml
+Cluster: example
+ > context: europe
+ > creating service account kubermatic-seed-account ...
+ > assigning cluster role kubermatic-seed-account-cluster-role ...
+ > reading auth token ...
+ > adding user example-kubermatic-service-account ...
+ > updating cluster context ...
+ > kubeconfig updated
+```
+
+The Seed resource then needs to reference the new kubeconfig Secret like so:
+
+```yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  name: kubeconfig-europe-west1
+  namespace: kubermatic
+type: Opaque
+data:
+  kubeconfig: <base64 encoded kubeconfig>
+
+---
+apiVersion: kubermatic.k8s.io/v1
+kind: Seed
+metadata:
+  name: europe-west1
+  namespace: kubermatic
+spec:
+  # these two fields are only informational
+  country: DE
+  location: Hamburg
+
+  # list of datacenters where this seed cluster is allowed to create clusters in
+  datacenters: []
+
+  # reference to the kubeconfig to use when connecting to this seed cluster
+  kubeconfig:
+    name: kubeconfig-europe-west1
+    namespace: kubermatic
+```
+
+Refer to the [Seed CRD documentation]({{< ref "../../concepts/seeds" >}}) for a complete example of the
+Seed CustomResource and all possible datacenters.
+
+Apply the manifest above in the master cluster and Kubermatic will pick up the new Seed and begin to
+reconcile it by installing the required Kubermatic components.
+
+### 4. Update DNS
```

Comment:
maybe for the DNS / LB setups a generic "Network Diagram" would be good. Something like the one we draw for some customer, but in a more generic way: https://drive.google.com/file/d/1Yv8-zR41U2EBPQ0merx1fDZenUDjtDtv/view?pli=1